pkg/updatechecker/updatechecker.go

```diff
@@ -752,19 +767,23 @@ func GetAvailableUpdates(kotsStore storepkg.Store, app *apptypes.App, license *k
 
 func isUpdateDeployable(updateCursor string, updates []upstreamtypes.Update) (bool, string) {
 	// iterate over updates in reverse since they are sorted in descending order
-	requiredUpdates := []upstreamtypes.Update{}
+	requiredUpdates := []string{}
 	for i := len(updates) - 1; i >= 0; i-- {
 		if updates[i].Cursor == updateCursor {
 			break
 		}
 		if updates[i].IsRequired {
-			requiredUpdates = append(requiredUpdates, updates[i])
+			requiredUpdates = append(requiredUpdates, updates[i].VersionLabel)
 		}
 	}
+	return canDeployVersion(requiredUpdates)
+}
+
+func canDeployVersion(requiredUpdates []string) (bool, string) {
```

Comment:
how about renaming this function and use it only if there are `requiredUpdates`?
```suggestion
func getNonDeployableCause(requiredUpdates []string) (bool, string) {
```
that way it's just about building the actual message.

Reply:
great idea!